userguide/getting-started/quick_start_with_kaui.adoc

```diff
@@ -0,0 +1,189 @@
+= Quick Start with Kaui
+:imagesdir: C:\_My Documents\FlowWritingLLC\Projects\Kill Bill\Documentation\killbill-docs\userguide\assets\img\getting-started
+
+Kaui is the back-office user interface for Kill Bill. This guide walks you through several tasks in Kaui. You will become familiar with moving around the application, and you'll learn some basic Kill Bill concepts:  
+
+. <<step1, Create a tenant>>
+. <<step2, Set up a simple catalog and plans>>
+. <<step3, Create an account>> 
+. <<step4, Add a payment method to the account>>
+. <<step5, Set up a subscription for the account>>
+. <<step6, View the generated invoice>>
+
+[NOTE]
+*Note:* You can also achieve these same results of this tutorial with the API. In fact, if you are going to have a front-end interface (such as a website), the API calls are what you use to communicate with Kill Bill (as opposed to using Kaui). For more information, see https://docs.killbill.io/latest/getting_started.html#_api_client[API Client] in the https://docs.killbill.io/latest/getting_started.html.html[_Getting Started_] guide.
+
+== Before You Begin
+
+Make sure you have installed Kill Bill and Kaui and launched them per the https://docs.killbill.io/latest/getting_started.html.html[_Getting Started_] guide. 
+
+== Additional Resources
+
+* https://docs.killbill.io/latest/userguide_kaui.html[_Kaui Guide_] 
+
+* https://killbill.github.io/slate[_API Reference_]
+
+* https://www.youtube.com/c/KillbillIoOSS[Kaui tutorials on YouTube]
+
+[[step1]]
+== Step 1. Create a Tenant
+
+Kill Bill supports multi-tenancy, where each tenant has its own data, configuration, and so forth. This section explains how to create a tenant the first time you log in to Kaui.
+
+. Log in to Kaui: 
++
+image::login-screen.png[width=400,align="center"]
++
+Kaui prompts you to create a tenant: 
++
+image::AddNewTenant-Blank.png[width=450,align="center"]
++
+. Fill in the fields:
++
+* *Name* refers to the tenant name.
+* For *API Key*, use `bob`, and for *API Secret*, use `lazar` in the rest of this guide.
++
+. Click the *Save* button. 
+
+After you create a tenant, the active tenant name appears in the upper right corner of the Kaui screen:
+
+image::select-tenant-name.png[width=850,align="center"]
+
```

Comment:
Now I'm confused - "bob" and "lazar" are the API key and secret. So you suggest we have "Bob Lazar" as the tenant name?

Reply:
Sorry about the confusion. In your `userguide/assets/img/getting-started/select-tenant-name.png` screenshot, the tenant name is `t_demo`. When I create a tenant called `bob` on my local machine, I see the following:
<img width="459" alt="image" src="https://user-images.githubusercontent.com/85998496/183339866-c27e4b50-6adb-45d3-aa71-9443f9eef982.png">
So, I thought it might be less confusing if we use a screenshot that has the tenant name as `bob`. But if it's too much trouble, we can leave it as it is.

Reply:
@reshmabidikar - This threw me for a loop for a while until I noticed that the AWS / Postman doc used "bob" as a tenant name. Is that why you wanted me to continue using it? I did change it to `bob` and adjusted the screenshots.